aem/build.gradle.kts

```diff
@@ -9,20 +7,31 @@ apply(from = rootProject.file("gradle/common.gradle.kts"))
 
 aem {
     environment {
+        docker {
+            init {
+                ensureDir(
+                        "httpd/cache",
+                        "httpd/logs"
+                )
```

Comment:
Why not having this section directly in httpd configuration below? Then you could omit httpd prefix.

Reply:
hmm but then we will force convention to have only files under .environment/httpd/...

I will think about it, thanks ;)

Reply:
but what if we have shared volumes?

then this common init section will be required

Reply:
yep connecting containers per this folder might be excellent idea as of containers are totally isolated
so that reflecting it here makes sense /